src/FMBot.Bot/Services/FriendsService.cs

```diff
@@ -31,6 +31,17 @@ public async Task<List<Friend>> GetFriendsAsync(ulong discordUserId)
         return friends;
     }
 
+    public async Task<List<Friend>> GetFriendedAsync(int userId)
+    {
+        await using var db = await this._contextFactory.CreateDbContextAsync();
+        
+        var friended = await db.Friends
+            .Where(f => f.FriendUserId == userId)
```

Comment:
Since you will be using the `FriendUser` object you need to include it; `.Include(i => i.FriendUser)`

Reply:
See my earlier comment, I suppose I should `.Include(i => i.User)`?

Reply:
Yes